_docs/deployment.md

```diff
@@ -17,21 +17,21 @@ subNav:
 permalink: /get-started/documentation/deployment.html
 ---
 
-The deploy directory in appserver.io distribution is the location where end users can place their deployment content (e. g. phar files) to have it deployed into the server runtime.
+The deploy directory in an appserver.io distribution is the location where end users can place their deployment content (e.g. phar files) to have it deployed into the server runtime.
 
-Users, particularly those running production systems, are encouraged to use the appserver.io AS management APIs to upload and deploy content.
+Users, particularly those running production systems, are encouraged to use the appserver.io application server (AS) management APIs to upload and deploy content.
 
 ## Deployment Modes
 
-The scanner only supports manual deployment mode, which means that you have to restart the server to process the deployment of your content. In this mode, the scanner will not attempt to directly monitor the deployment content and decide if or when the end user wishes the content to be deployed or undeployed. Instead, the scanner relies on a system of marker files, with the users addition or removal of a marker file serving as a sort of command telling the scanner to deploy, undeploy or redeploy content.
+The scanner only supports manual deployment mode, which means that you have to restart the server to process the deployment of your content. In this mode, the scanner will not attempt to directly monitor the deployment content and decide if or when the end user wishes the content to be deployed or undeployed. Instead, the scanner relies on a system of marker files, where the user's addition or removal of a marker file serves as a sort of command telling the scanner to deploy, undeploy or redeploy content.
 
-It is also possible to copy your unzipped content directly into the webapps folder. After restarting the web server your content will then be deployed without having any impact on the deployment scanner because only zipped (.phar) content will be recognized.
+It is also possible to copy your unzipped content directly into the webapps folder. After restarting the web server, your content will then be deployed without having any impact on the deployment scanner, because only zipped (.phar) content will be recognized.
```

Comment:
This is technically incorrect from our side! :( 
Restarting only the webserver (although not possible) does not do anything, the complete appserver has to be restarted (at least by now).
Might you change it?